### PR TITLE
Show payout methods for accounts without hosts

### PR DESCRIPTION
**Pull request by Betree**
Fix https://github.com/opencollective/opencollective/issues/5201

### CODE REVIEW

File: components/expenses/PayoutMethodSelect.js [hunk 1]
```diff
@@ -202,16 +202,19 @@ class PayoutMethodSelect extends React.Component {
     title: this.getPayoutMethodTitle(pm),
   });
 
-  getPayerHostSupportedPayoutMethods = () => {
-    return this.props.collective.host?.supportedPayoutMethods || EMPTY_ARRAY;
-  };
-
-  getOptions = memoizeOne((payoutMethods, payee) => {
-    const payerHostSupportedPayoutMethods = this.getPayerHostSupportedPayoutMethods();
+  getOptions = memoizeOne((host, payoutMethods, payee) => {
```
Comment (Betree) [inline components/expenses/PayoutMethodSelect.js:205]: Moving the host from `this.props` to arg as the function is memoized

**Comment by znarf** [comment]
Would that be complicated to have structured Bank Transfer too?

I assume it should not be dependent on having Wise connected (we have proper code to support this case).

**Comment by Betree** [comment]
> Would that be complicated to have structured Bank Transfer too?
> 
> I assume it should not be dependent on having Wise connected (we have proper code to support this case).

I just added support for this, it was fairly simple given that most parts of the code were already handling the case for missing host.

**Review by znarf — APPROVED** [review]
Looks good!

**Pull request merged**